Linux.md

```diff
@@ -0,0 +1,37 @@
+# Linux
+
+![alt text][logo]
+
+[logo]: https://upload.wikimedia.org/wikipedia/commons/a/af/Tux.png "Linux mascot - Tux"
+
+Linux in simple terms is an operating system. It is very similar to operating systems like Windows and Mac OS.
+[Linus Torvalds](https://en.wikipedia.org/wiki/Linus_Torvalds) built the core of the Linux operating system, known as the kernel. A kernel alone does not make an operating system, but Stallman's [GNU](https://en.wikipedia.org/wiki/GNU) tools were from a project to create an operating system as well--a project that was missing a kernel to make Stallman's operating system complete. Torvalds' matching of GNU tools with the Linux kernel marked the beginning of the Linux operating system as it is known today.
+
+### What is kernel?
+
+Kernel is the central part of an operating system that is responsible for interfacing all the applications down to the physical hardware. All operating systems have kernels.
```

Comment:
This sounds vague. Could you take examples and describe what is a part of kernel and what is not?

For instance, `vim` comes pre-installed with most Linux distro, but not part of the kernel. Also, point to GitHub repo on Kernel.